src/Eloquent/Builder.php

```diff
@@ -153,14 +153,6 @@ public function decrement($column, $amount = 1, array $extra = [])
         return parent::decrement($column, $amount, $extra);
     }
 
-    /**
-     * @inheritdoc
-     */
-    public function chunkById($count, callable $callback, $column = '_id', $alias = null)
```

Comment:
I'm not familiar with `chunkById()`, but https://laravel.com/docs/10.x/queries#chunking-results states:

> If you plan to update the retrieved records while chunking, it is always best to use the chunkById method instead. This method will automatically paginate the results based on the record's primary key:

Looking at the test below, I'm not sure where the primary key is being overwritten. How do we know an `_id` field isn't being created internally? Or does the User model in the test suite define `name` as a primary key mapped to the `_id` database field?

Reply:
The primary key is set in the model. [`_id` is the default for MongoDB models](https://github.com/GromNaN/laravel-mongodb/blob/master/src/Eloquent/Model.php#L39).

This key is used in Laravel's [`chunckById`](https://github.com/laravel/framework/blob/54421e4734db69b0dd54efdb810ce294c0e774ae/src/Illuminate/Database/Concerns/BuildsQueries.php#L114-L116).